source/setup.py

```diff
@@ -1,70 +1,35 @@
 # -*- coding: UTF-8 -*-
-#setup.py
-#A part of NonVisual Desktop Access (NVDA)
-#Copyright (C) 2006-2018 NV Access Limited, Peter Vágner, Joseph Lee
-#This file is covered by the GNU General Public License.
-#See the file COPYING for more details.
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2006-2022 NV Access Limited, Peter Vágner, Joseph Lee
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
 
 import os
 import sys
 import copy
 import gettext
 gettext.install("nvda")
 from setuptools import setup
-import py2exe as py2exeModule

```

Comment:
While this import appears unused it is unfortunately needed. To work py2exe has to monkey patch distutils, and this monkey patch is applied when importing py2exe for the first time. Our build still works regardless, because we import `py2exe.distutils_buildexe` below, but in this case it seems better to be explicit and re-introduce the import of `py2exe` with a appropriate NOQA and a comment explaining that `distutils` is monkey patched on import.

Reply:
Thanks for pointing this out, added a comment